eng/build-job.yml

```diff
@@ -140,12 +140,17 @@ jobs:
           KeyVaultName: EngKeyVault
           SecretsFilter: 'dotnetfeed-storage-access-key-1,microsoft-symbol-server-pat,symweb-symbol-server-pat'
 
+    - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
+      - script: ./build-packages.sh -BuildArch=$(archType) -BuildType=$(buildConfigUpper) $(crossPackagesArg) -OfficialBuildId=$(Build.BuildNumber) $(portableBuildArg)
+        displayName: Build packages
+
```

Comment:
Nit: could you remove the line between the build-packages.sh/cmd steps and add a comment above the two to make it visually clear that they're conceptually the same step?

Reply:
Sure